huaweicloud/services/aad/resource_huaweicloud_aad_forward_rule.go

```diff
@@ -0,0 +1,256 @@
+package aad
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/aad/v1/rules"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+// ResourceForwardRule is the imple of huaweicloud_aad_forward_rule
+func ResourceForwardRule() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceForwardRuleCreate,
+		ReadContext:   resourceForwardRuleRead,
+		UpdateContext: resourceForwardRuleUpdate,
+		DeleteContext: resourceForwardRuleDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceForwardRuleImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
```

Comment:
this parameter is missing in the doc
should it be ForceNew?

Reply:
This parameter has been removed.